pkg/cmd/gpg-key/add/http.go

```diff
@@ -25,6 +25,7 @@ func gpgKeyUpload(httpClient *http.Client, hostname string, keyFile io.Reader) e
 
 	payload := map[string]string{
 		"armored_public_key": string(keyBytes),
+		"name":               title,
```

Comment:
If title is blank will the request get rejected by the server? Or will the key still be added? Will there be an automatically generated name?

Reply:
As per my local testing, if the title is blank, this does not get rejected by the server, and the key is properly added without a title. Nothing is auto-generated by the backend.

If we're scared about the backend changing its behavior on this, maybe we could only include the name field conditional on title being nonempty, to be extra safe?

Just like SSH keys auto-generate titles, the backend _could_ also auto-generate titles for GPG keys based on the time they were added and the email address associated with the key. Then, there would be no difference between how SSH and GPG would need to be handled. But alas, we live in a world where GPG keys sometimes have titles but SSH keys always have titles.